ccd-config-generator/build.gradle

```diff
@@ -10,11 +10,22 @@ repositories {
     maven { url 'https://pkgs.dev.azure.com/hmcts/Artifacts/_packaging/hmcts-lib/maven/v1' }
 }
 
+configurations {
```

Comment:
Sorry to be a pain but can we please follow the config diffing based approach to testing? per the readme - 

Please ensure the [tests](https://github.com/hmcts/dtsse-ccd-config-generator/blob/master/ccd-config-generator/src/test/java/uk/gov/hmcts/ccd/sdk/E2EConfigGenerationTests.java) have been updated to cover any new or altered functionality. The tests are based on comparing the generated JSON output to [expected](https://github.com/hmcts/dtsse-ccd-config-generator/tree/master/ccd-config-generator/src/test/resources/CARE_SUPERVISION_EPO) - expected output should be added to/modified as necessary.

Per the issue you raised this should work well since you've already identified the correct output that you seek - https://github.com/hmcts/dtsse-ccd-config-generator/issues/554